### ISSUE TITLE
Sentiment Scale: Component should not have `aria-disabled` attribute set when not not disabled.

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2377]
Check the Accessibility Incomplete tab at:
https://modus-web-components.trimble.com/?path=/story/components-sentiment-scale--default

The generated DOM of the component is:
```html
<div class="sentiment-scale-container" aria-disabled="" aria-label="" role="group">
```

`aria-disabled` should be removed.

![image](https://github.com/trimble-oss/modus-web-components/assets/1212885/16615861-6238-4373-9ae3-c570d4bd1a86)
